cmd/fleetctl/apple_mdm.go

```diff
@@ -44,124 +52,137 @@ func appleMDMCommand() *cli.Command {
 	}
 }
 
-func appleMDMSetupCommand() *cli.Command {
+func generateCommand() *cli.Command {
 	return &cli.Command{
-		Name:  "setup",
-		Usage: "Setup commands for Apple MDM",
+		Name: "generate",
+		Flags: []cli.Flag{
+			configFlag(),
+			contextFlag(),
+			debugFlag(),
+		},
 		Subcommands: []*cli.Command{
-			appleMDMSetupSCEPCommand(),
-			appleMDMSetupAPNSCommand(),
-			appleMDMSetupDEPCommand(),
+			generateMDMAppleCommand(),
 		},
 	}
 }
 
-func appleMDMSetupSCEPCommand() *cli.Command {
-	// TODO(lucas): Define workflow when SCEP CA certificate expires.
-	var (
-		validityYears      int
-		cn                 string
-		organization       string
-		organizationalUnit string
-		country            string
-	)
+func generateMDMAppleCommand() *cli.Command {
 	return &cli.Command{
-		Name:  "scep",
-		Usage: "Create SCEP certificate authority",
+		Name:  "mdm-apple",
```

Comment:
In https://github.com/fleetdm/fleet/pull/8786/files#diff-c83f7602b8ed12b771f57bf632370b0f5da9a70d7da23c0d146c436709ce1e74R1061-R1063 we're adding support for `mdm_apple` and `mdm-apple` as an alias, I don't have a strong opinion on whether we should do it or not, but we should definitely be consistent.

Should we change things here or in the linked PR? any opinions? cc: @mna

Reply:
Why are we adding aliases for these commands? Aliases are usually added to "shorten" long commands and make them more convenient to type. Additionally, it doesn't look like we use underscores in any other subcommands in fleetctl, nor do we use aliases.

Reply:
We do use some aliases, I added the `mdm_apple` one for consistency with those:

```
   enroll_secret, enroll_secrets, enroll-secret, enroll-secrets  Retrieve the osquery enroll secrets
   user_roles, user_role, ur                                     List global and team roles for users
```

@noahtalerman what do you think? Should we have them or remove them? For `user_roles` it looks like we don't even have the `user-role`/`user-roles` versions.

Reply:
> what do you think?

@mna @michalnicp I don't have a strong preference. Up to you.

That said, I think if other commands have aliases, we should be consistent and include aliases here.

Reply:
I think when there's a dash/underscore, it's nice UX to have an alias to the other option, as it's easy to forget which one it is. I'd also add the `user-roles`/`user-role` aliases.

Reply:
Fixed in 669a8a1